lib/routes/json_api_routes.rb

```diff
@@ -61,13 +61,19 @@ def media_resources(*names)
       end
     end
 
+    def json_api_default_opts
+      @json_api_default_opts ||= {
+        except: %i(new edit),
+        constraints: { id: VALID_IDS },
```

Comment:
Unrelated, but shouldn't https://github.com/zooniverse/Panoptes/pull/2515/files#diff-6c7cf9697e69227d67c03ea7e9bd79cdL3 be `/[0-9]+/` rather than `*`?

Reply:
I believe you are correct. I'll issue a new PR to fix this in isolation.

Reply:
#2517